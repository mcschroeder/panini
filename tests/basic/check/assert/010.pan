assert : { b:𝔹 | b = true } → 𝟙

f10 : unit
f10 = assert true

f11 : {v:unit|true}
f11 = assert true

f12 : {v:unit|v=unit}
f12 = assert true
