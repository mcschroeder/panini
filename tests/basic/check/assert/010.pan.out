f10 : 𝟙
f11 : {v:𝟙 | true}
f12 : {v:𝟙 | v = unit}